src/NuGet.Clients/PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -209,17 +302,20 @@ public UIActionEngine(ISourceRepositoryProvider sourceProvider, NuGetPackageMana
                 // preview window
                 if (uiService.DisplayPreviewWindow)
                 {
-                    var shouldContinue = uiService.PromptForPreviewAcceptance(results);
+                    /*var shouldContinue = uiService.PromptForPreviewAcceptance(results);
                     if (!shouldContinue)
                     {
                         return;
-                    }
+                    }*/
                 }
```

Comment:
Remove this whole block rather than comment it out, or put it under an ifdef if required for debugging